app/tests/integration/nimbus/test_experimenter.py

```diff
@@ -0,0 +1,62 @@
+import time
+
+import pytest
+from nimbus.pages.home import HomePage
+from nimbus.pages.summary import SummaryPage
+
+
+@pytest.mark.run_once
+def test_archive_experiment(
+    selenium,
+    base_url,
+    default_data,
+    create_experiment,
+):
+    selenium.get(base_url)
+    home = HomePage(selenium, base_url).wait_for_page_to_load()
+
+    # Create experiment
+    summary_page = create_experiment(selenium, home, default_data)
+
+    # Archive it on the summary page
+    summary_page.archive_button.click()
+
+    # Check it's archived on the summary page
+    time.sleep(2)
+    assert summary_page.archive_label.is_displayed()
```

Comment:
Should this retry a couple in a loop at all in case the label isn't available within ~2s?

Reply:
Should this also have some error text if the assertion fails? Something like https://github.com/mozilla/experimenter/blob/19298d682fcbb5e38cc8b49fd0def8d414764b5d/app/tests/integration/nimbus/test_e2e_create_experiment.py#L125

But not sure that it's a pattern we use globally in the test assertions, so not a huge deal.

Reply:
Ah I learned an even better pattern for all of this!

```py
        self.wait.until(
            EC.presence_of_all_elements_located(self._archive_label_locator),
            message="Summary Page: could not find archive label",
        )
```